src/librustc_typeck/astconv.rs

```diff
@@ -2358,10 +2358,11 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
                     break;
                 }
             }
-            for binding in segment.generic_args().bindings {
+
+            // only emit the first error to avoid overloading the user with error msgs
```

Comment:
```suggestion
            // Only emit the first error to avoid overloading the user with error messages.
```
We generally prefer full sentences for consistency.
r=me after fixing.